tests/Unit/Helpers/Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/TestHelpers.hpp

```diff
@@ -105,6 +124,13 @@ void check_impl(
  * DampingFunction is arbitrary, but should generally be descriptive (e.g.
  * 'gaussian_plus_constant') of the DampingFunction.
  *
+ * The input parameter `function_of_time_name` is the name of the FunctionOfTime
+ * that will be included in the FunctionsOfTime passed to the DampingFunction's
+ * call operator. For time-dependent DampingFunctions, this parameter must be
+ * consistent with the FunctionOfTime name that in_gh_damping_function's call
```

Comment:
ticks around `in_gh_damping_function` would make this easier to read

Reply:
Good catch! I just pushed and squashed